.github/workflows/build-push-docker-image.yml

```diff
@@ -62,10 +118,45 @@ jobs:
           # cache-to: type=gha,mode=max
 
       - name: Run Tests
+        if: ${{ inputs.skip_tests != true || needs.get_changed_files.outputs.should_execute_tests == true }}
         run: |
           docker run ${{ inputs.build_tag }} npm run test:js:ci
           docker run ${{ inputs.build_tag }} npm run test:ts:ci
 
+      - name: Notify Slack Channel
+        if: ${{ inputs.skip_tests != true || needs.get_changed_files.outputs.should_execute_tests == true }} && failure()
+        id: slack
+        uses: slackapi/slack-github-action@v1.25.0
+        continue-on-error: true
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+          PROJECT_NAME: 'Rudder Transformer'
+        with:
+          channel-id: ${{ secrets.SLACK_RELEASE_CHANNEL_ID }}
+          payload: |
+            {
+              "text": "*<${{inputs.build_type}} release tests failed>*\nCC: <@U03KG4BK1L1> <@U024YF8CR53> <@U03JWRHH1NJ>",
```

Comment:
should be notified to #integration-dev & tag @integration-eng group (<!subteam^GROUP_ID>)